video_analysis_ui.py

```
@@ -0,0 +1,487 @@
+            st.markdown("#### ⏱️ Processing Time")
+            duration = llm_analysis['processing_duration_seconds']
+            st.metric("Duration", f"{duration:.2f}s")
+    
+    # Topics
+    if llm_analysis.get('topics'):
+        st.markdown("#### 🏷️ Detected Topics")
+        topics = llm_analysis['topics']
+        if isinstance(topics, str):
+            try:
+                topics = json.loads(topics)
+            except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)